holisticai/explainability/metrics/feature_importance/local_importance/_lime_importance_spread.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+
+from holisticai.explainability.metrics.feature_importance.utils import importance_spread
+
+
+def feature_importance_spread_lime(
+    feature_importance, conditional_feature_importance, lime_importance
+):
+    if lime_importance == "dataset":
+        metric_name = "Dataset"
+        imp_spread = {
+            "Global": feature_importance.groupby("Sample Id").apply(
+                lambda df: importance_spread(
+                    df.set_index("Feature Label")["Importance"], divergence=True
+                )
```

Comment:
imp spread comment 1/2 --> Here, for each sample, you get an importance spread metric (so say you have 50 samples that gives you 50 importance spread numbers)